src/Microsoft.DotNet.Build.Tasks/PackageFiles/Roslyn.Common.props

```diff
@@ -4,8 +4,11 @@
     <RoslynVersion>2.0.0-rc</RoslynVersion>
     <RoslynPackageName>Microsoft.Net.Compilers</RoslynPackageName>
     <RoslynTargetsPath>$(ToolRuntimePath)</RoslynTargetsPath>
-    <CSharpCoreTargetsPath Condition="'$(CSharpCoreTargetsPath)' == ''">$(RoslynTargetsPath)$(RoslynPackageName)\$(RoslynVersion)\tools\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
-    <VisualBasicCoreTargetsPath Condition="'$(VisualBasicCoreTargetsPath)' == ''">$(RoslynTargetsPath)$(RoslynPackageName)\$(RoslynVersion)\tools\Microsoft.VisualBasic.Core.targets</VisualBasicCoreTargetsPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
```

Comment:
>If we set RoslynTargetsPath the same way as CoreFx does, doesn't that mean we're always running off of the netcore Roslyn compiler?

Isn't these equivalent? I personally would start there and see we run into any problems.